src/mainwindow.cpp

```diff
@@ -25,8 +24,9 @@
 MainWindow::MainWindow(QWidget *parent) :
     QMainWindow(parent),
     ui(new Ui::MainWindow),
-    m_monitor(new Monitor),
-    m_plotterView(nullptr) {
+    m_plotterView(nullptr),
+    m_readFirstPass(true),
+    m_monitorVerticalScrollBarGrabbing(false) {
 
     ui->setupUi(this);
     QList<qint32> baudRates {QSerialPort::Baud1200, QSerialPort::Baud2400, QSerialPort::Baud4800, QSerialPort::Baud9600, QSerialPort::Baud19200, QSerialPort::Baud38400, QSerialPort::Baud57600, QSerialPort::Baud115200};
```

Comment:
You can make this an enum (assuming these baud rate values are integers)

Reply:
Well so QSerialPort::BaudRate is already an enum (and the values are integers, qint32 to be specific), but how do you iterate over the values of an enum? I basically asked Deep what baud rates we'll use and put them into a qlist for easy enumeration

Reply:
Fair point, could just make it a static array.